assets/js/googlesitekit/modules/create-module-store.js

```diff
@@ -79,12 +88,19 @@ export const createModuleStore = ( slug, {
 			settingSlugs,
 		} );
 
+		const submitChangesStore = createSubmitChangesStore( {
+			storeName,
+			submitChanges: submitChanges || makeDefaultSubmitChanges( slug, storeName ),
+			validateCanSubmitChanges: validateCanSubmitChanges || makeDefaultCanSubmitChanges( storeName ),
+		} );
+
 		// to prevent duplication errors during combining stores, we don't need to combine
 		// Data.commontStore here since settingsStore already uses commonActions and commonControls
 		// from the Data.commonStore.
 		combinedStore = Data.combineStores(
 			notificationsStore,
 			settingsStore,
+			submitChangesStore,
```

Comment:
The `else` clause (where `settingSlugs` is `undefined`) is missing the `submitChangesStore`. Per the ACs, this partial store should be part of _every_ module store, regardless of whether or not it has settings. Only the defaults should differ (in case of no settings, rely on the regular defaults from `createSubmitChangesStore`).

Reply:
Updated.